app/views/teacher_training_adviser/steps/_has_teacher_id.html.erb

```diff
@@ -1,6 +1,16 @@
-<% @page_title = "Do you have your previous teacher reference number?" %>
-
-<%= f.govuk_radio_buttons_fieldset(:has_id, inline: true, legend: { tag: "h1" }) do %>
+<%= f.govuk_radio_buttons_fieldset(:has_id, inline: true, legend: { tag: "h1", text: "Do you have a teacher reference number (TRN)?" }) do %>
   <%= f.govuk_radio_button :has_id, true, label: { text: 'Yes' }, link_errors: true %>
   <%= f.govuk_radio_button :has_id, false, label: { text: 'No' }, link_errors: true %>
 <% end %>
+
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      What is a teacher reference number (TRN)?
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    A TRN is a 7 digit number that you get if you hold qualified teacher status in England or Wales or you contribute to the Teachers' Pension Scheme.<br><br>
```

Comment:
A TRN is a 7 digit number you automatically get if you:

- hold qualified teacher status (QTS) in England 
- hold early years teacher status (EYTS) in England
- hold a national professional qualification (NPQ)
- are working towards QTS, EYTS or a national professional qualification (NPQ)
- contribute to the Teachers’ Pensions scheme in England 
- have a relevant restriction in relation to teaching in England